src/function_wrappers.ts

```diff
@@ -122,6 +131,37 @@ const wrapHttpFunction = (execute: HttpFunction): RequestHandler => {
   };
 };
 
+const wrapHttpAsyncFunction = (
+  userFunction: OpenFunction,
+  context: OpenFunctionContext
+): RequestHandler => {
+  const ctx = OpenFunctionRuntime.ProxyContext(context);
```

Comment:
I'm not sure if Nodejs can do this, but function-framework-go put HTTP request/response in function context for HTTP sync functions, and user can do whatever they used to do with the HTTP request/response inside their function `userFunction(ctx, data)`

[userFunction(ctx, req.body)](https://github.com/OpenFunction/functions-framework-go/blob/main/runtime/knative/knative.go#L62)

Reply:
Surely we can do this in Node.js, I haven't noticed this `SyncRequest` object in context, would refactor this part to align with Go FF implemenatation.